sklearn/manifold/t_sne.py

```diff
@@ -639,29 +640,28 @@ def _fit(self, X, skip_num_points=0):
             raise ValueError("'method' must be 'barnes_hut' or 'exact'")
         if self.angle < 0.0 or self.angle > 1.0:
             raise ValueError("'angle' must be between 0.0 - 1.0")
+        if self.method == 'barnes_hut':
```

Comment:
pretty sure this doesn't belong here.

Reply:
Yeah, messed up with the branch. Will create another PR